source/adminguide/host_and_storage_tags.rst

```diff
@@ -49,6 +49,37 @@ To explain the behavior of host tags, some examples will be demonstrated with tw
     * Offering: h3
     None of the hosts have compatible tags and it will not be possible to deploy a VM with the offering. However, CloudStack ignores this behavior when a host is manually selected.
 
+.. _strict-host-tags:
+Strict Host Tags
+-----------------
+During certain operations, like changing compute offering, and starting or live
+migrating an instance to a specific host, CloudStack will ignore the host tags,
+which can lead to issues where a instance starts on an undesired host.
```

Comment:
We should (?) reword this to explain that "ignoring tags" is mostly (or only?) done when Admin does some operations (and this is by design/desired in most cases) - however we are adding an add-on feature to force tags checks by this global setting.